src/otx/algorithms/classification/configs/deit_tiny/model.py

```diff
@@ -18,3 +18,5 @@
 
 optimizer = dict(_delete_=True, type="AdamW", lr=0.01, weight_decay=0.05)
 optimizer_config = dict(_delete_=True)
+
+custom_hooks = [dict(type="AdaptiveRepeatDataHook")]
```

Comment:
Both of hooks still alive at the training phase. LazyEarlystop, AdaptiveRepeatData
![image](https://github.com/openvinotoolkit/training_extensions/assets/80735344/9612433d-7ccb-46fa-a58f-4d39a0f4eb2e)

Reply:
I checked that `LazyEarlystop` configuration is override but it's newly patched at `patch_early_stopping()`. It can be refined later but anyway it's ok now.